src/backend/dx11/src/debug.rs

```diff
@@ -73,7 +47,6 @@ impl Drop for DebugScope {
     }
 }
 
-#[cfg(debug_assertions)]
 pub fn debug_marker(context: &ComPtr<d3d11::ID3D11DeviceContext>, args: fmt::Arguments) {
```

Comment:
this should probably just get an `&str`. No point to receive the arguments if all we do is just creating a `String` for them, anyway